src/feature/urltopdf/index.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const aws = require(`./lib/aws`);
+const fs = require('fs');
+const scrape = require('website-scraper');
+const PuppeteerPlugin = require('./lib/plugin');
+const { zip } = require('zip-a-folder');
+const sendMsg = require('./lib/sqsMessage')
+
+
+exports.handler = async (event, context) => {
+
+  let event = JSON.stringify(event.Records[0].body);
```

Comment:
Please don't redeclare function parameters. It could cause confusion with respect to what variable you're referring to and is really not necessary. Additionally, I don't see the need for you to stringify a JSON object in code. If you need to check whether an attribute is present or not you can use a ternary operator checking for whether the attribute is truthy. You might also want to check whether the records array is null or empty

```suggestion
  if (!event.Records || event.Records.length === 0) {
    throw "No event records";
  }
  const { body } = event.Records[0];
```